src/DetailsView/reports/components/automated-check-issue-details-instances.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as classNames from 'classnames';
+import * as React from 'react';
+import { NamedSFC } from '../../../common/react/named-sfc';
+import { DecoratedAxeNodeResult } from '../../../injected/scanner-utils';
+
+export interface AutomatedChecksIssueDetailsInstancesProps {
+    nodeResults: DecoratedAxeNodeResult[];
+}
+
+const createTableRow = (label, content, rowKey) => {
+    const contentIsSnippet: boolean = label === 'Snippet';
+    const contentStyling = classNames({
```

Comment:
why not use the `rowKey` as `className` also (so both, `key` and `className`), you already have the proper stylings for the snippets so that should work and it's less complex... and you avoid being too much specific with this check:
```typescript
const contentIsSnippet: boolean = label === 'Snippet';
```

Reply:
As discussed offline, made the change that seemed reasonable.